### PR TITLE
Add changelog property on relevant services

### PR DESCRIPTION
**Pull request by rambobinator**
Regarding Appnexus documentation:
https://wiki.appnexus.com/display/api/Change+Log+Services#ChangeLogServices-getdetails
Relevant services are:
- insertion-order
- line-item
- campaign
- profile

I had to rename service parameter to _service since change-log endpoint use it as needed argument.

I also add profile property the same way to keep services creation on a single function instead of redefining class everywhere.

### CODE REVIEW

File: appnexus/client.py [hunk 1]
```diff
@@ -216,49 +216,51 @@ def base_url(self):
...
-                 "UsergroupPattern", "VisibilityProfile", "Report"]
+                 "MobileAppInstance", "MobileAppInstanceList",
+                 "MobileAppStore", "OperatingSystemExtended",
+                 "OperatingSystemFamily", "OptimizationZone", "Package",
+                 "PackageBuyerAccess", "PaymentRule", "Pixel", "Placement",
+                 "PlatformMember", "PostalCode", "Profile", "ProfileSummary",
+                 "Publisher", "Region", "Report", "ReportStatus", "Search",
+                 "Segment", "Site", "TechnicalAttribute", "Template",
+                 "ThirdpartyPixel", "User", "UsergroupPattern",
+                 "VisibilityProfile"]
```
[the remark this answers is not in the document]
Comment (rambobinator) [reply appnexus/client.py]: I sorted the entire list, and it wasn't well ordonned.

**Comment by rambobinator** [comment]
@ramnes i made the changes

**Comment by ramnes** [comment]
@romas, can you please also add an example of usage inside the README? You can also change `service` to `service_name` there, on the custom representation code block.

**Comment by rambobinator** [comment]
@ramnes done

**Comment by ramnes** [comment]
Thanks!

**Pull request merged**